index.bs

```diff
@@ -373,6 +373,22 @@ The <dfn attribute for="Document">onprerenderingchange</dfn> attribute is an [=e
 
 As is customary for [[HTML]], the definition of {{Document/prerendering}} would be located in another section of the spec; we'd place it in the new section introduced below:
 
+<h3 id="performance-navigation-timing-extension">Extensions to the {{PerformanceNavigationTiming}} interface</h3>
+
+We'd modify the definition of {{PerformanceNavigationTiming}} as follows:
+
+<pre class="idl">
+  partial interface PerformanceNavigationTiming {
+      readonly attribute DOMHighResTimeStamp activationStart;
+  };
+</pre>
+
+When getting the value of the <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> attribute, run the following steps:
+
+1. If the current [=browsing context=] has not been [=prerendering browsing context/activate|activatted=], return a {{DOMHighResTimeStamp}} with a time value equal to zero.
```

Comment:
This is a bit subtle with descendent browsing contexts. Only the top-level browsing context is the one where the activate steps run. And it's not clear at which point we call it "activated".

I think we need to set a timestamp at Step 5 "Prompt to unload" of the activate steps, and read that timestamp here.

Reply:
Actually I'm not sure what the intent is with iframes. Does each iframe's navigation entry also have a populated activationStart? For same-origin iframes, we plan to load them as normal; for cross-origin iframes, the load is delayed until after activation.

Reply:
Done.
Introduced Document's "activation start time", and the local variable "activation start time".